### ISSUE TITLE
Help user add Unee-T icon on the home screen of mobile phone 

### ISSUE THREAD

**Issue by franck-boullier** [unee-t/frontend#428]
# The problem:

We want to increase Unee-T usage.
To do this we need to make Unee-T more accessible for users.
One way to facilitate access is to create a shortcut to Unee-T on the home screen of the mobile phone of the user.

We should add a one click option in Unee-T  so that the user can create this shortcut.

This link should be 
- easily accessible in the Unee-T interface
- prompted as a tip to the user once he/she has logged in to Unee-T (for example as a tip when the user logs in for the second time)

@kiatlim any suggestion?

**Comment by kiatlim** [comment]
Is Unee-T considered a Progressive Web App (PWA)?

Found something for Chrome on Android for PWA, its like an auto prompt it seems:
https://developers.google.com/web/fundamentals/app-install-banners/

For iOS wise, an overlay something like this could be done to prompt users:
![ath-preview](https://user-images.githubusercontent.com/641679/43688320-105e2746-9919-11e8-976e-d9ecf0b15e4e.png)

**Comment by kaihendry** [comment]
AFAIK there is no one click solution, you have to implicitly add to Home screen from the mobile browser. Install banners AFAIK do not know if the App is already installed / i.e. has an icon on the home screen. Perhaps Android is better, though I'm a IOS Safari user.

I think going down the road to thinking that the App requires an icon on the home screen to be considered "accessible" can be misleading.

I've heard of research (need to find reference) showing that users are still more likely to search for the app in the browser than find the App on one of their home screens.

Perhaps focusing on making a search term like "uneet" load our App first is a better approach to helping users find and use the application.

**Comment by kiatlim** [comment]
Yes there's no one click solution, user still need to add it to their home screen manually and it doesn't know if there was one already. 

But after adding, searching for it on iOS (not searching on browser) does show up Unee-T on the results.  
![img_9873-2](https://user-images.githubusercontent.com/641679/43697169-c25a7fde-9974-11e8-9618-9489539fb54a.png)

Since searching "uneet" generates lots of results of others, we can consider to run google adwords to surface Unee-T up, provided the cost per click are not too costly and in the long run if we need to run ads to compete, this would just become advertising / user acquisition costs.

**Comment by franck-boullier** [comment]
# Clarification:

@kiatlim: from a UI/UX perspective, when/where do you suggest that we add a prompt to the user to let him/her know that he/she can 'Create a shortcut on your homescreen"?

The assumption here is that the user is already on case.unee-t.com (either he/she has received an invite or he/she has created an account by self registering).

# The objective:

Make sure we offer that option to the user when we have the highest chance that the user "converts" i.e the user accepts to install an icon/shortcut on his/her mobile phone.

This "highest conversion ratio moment" will probably be different for:
- Self registered users
- Invited users

# Idea/Suggestion:
 
We want to make sure the user is committed to Unee-T and will need an easy access to Unee-T in the future.
This is probably not right after the user logs in for the first time.
Maybe when the user starts a conversation ---> when the user creates a message or invites another user to a case?

@kiatlim let me know your thoughts

**Comment by kiatlim** [comment]
@franck-boullier your suggestion is good, here's where I would add the prompt:

**For new user:**
1. After the user first created a unit
2. After the user first created a case
3. After the user first created an inspection report

**For invited user:**
1. After he has put in his first response to the message

**Comment by franck-boullier** [comment]
@kiatlim I like these ideas: can you add some mockups so we can have a better idea of how the flow will work?

Thanks!
